### DISCUSSION TITLE
Two Factor Auth setup missing country.

### DISCUSSION THREAD

**Discussion by APavlov2** [community/community#4677] [Other Feature Feedback, Questions, & Ideas]
I recently had to set up two-factor auth for my account, and I didn't want to use an app as the app is Device specific while phone numbers can be easily recovered in case of a lost or stolen phone.

That being said, my current number has the +389 prefix, which is for North Macedonia. 
This option is missing in the drop-down menu.

Are there plans to add it any time soon?

**Comment by abderrazzaq-laanaoui** [comment]
I faced the same issue, I am from morocco (prefix: +212).

**Comment by dragonsxslayer** [comment]
same problem
